target/common/common.mk

```diff
@@ -24,9 +24,13 @@ TB_SRCS   := $(wildcard ${ROOT}/hw/ip/test/*.sv)
 TB_DIR    := ${ROOT}/target/common/test
 
 VSIM_BENDER   += -t test -t rtl -t simulation -t vsim
-VSIM_SOURCES   = $(shell ${BENDER} script flist ${VSIM_BENDER} | ${SED_SRCS})
```

Comment:
I would keep the `XXX_SOURCES` variables here, since they are used the same also in Occamy. Is there a conflicting reason to move them to the target-specific Makefile?